### PR TITLE
Extract BIDS bold task names as scan ids in fMRIQC

### PR DESCRIPTION
**Pull request by wtriplett**
Suggested to address #21

### CODE REVIEW

**Comment by chrisfilo** [comment]
Would this give the same label to multiple runs?

**Comment by wtriplett** [comment]
@chrisfilo: I don't think so, the code extracts the run number portion of the filename if its present, but if not it just grabs the task name.

**Comment by chrisfilo** [comment]
I see. So 'taskinfo' is matching run number (plus 'run-' prefix) not the
task name as the variable name suggest?

**Comment by wtriplett** [comment]
@chrisfilo good point. Ideally if its a BIDS dataset we could break out the other qualifiers (acq-???, rec-???) into separate columns for completeness. The lesser used qualifiers would be empty more often than not, but at some point this is going to come up. Maybe #21 could be merged into issue #8 regarding multi-session datasets in favor of more comprehensive BIDS support in general?

In the shorter term having task name included in the CSV file and plots is critical for datasets with multiple tasks. 

@oesteban I'm happy to keep working on this and issue #8 -- let me know what you'd prefer.

File: mriqc/utils.py [hunk 1]
```diff
@@ -58,10 +58,15 @@ def gather_bids_data(dataset_folder, subject_inclusion=None, scan_type=None):
         if get_func:
             functional_scans = sorted(glob(op.join(
                 dataset_folder, subject_id, "func",
-                "%s_*bold.nii.gz" % subject_id, )))
+                "%s_*bold*.nii.gz" % subject_id, )))
 
         for i, scan in enumerate(functional_scans):
-            scid = 'func_%d' % (i+1)
+            taskinfo = re.search(r'.+?task-(([^_]+)(_run-([0-9]+))?)_bold.nii.gz$', scan)
+            if taskinfo == None:
```
Comment (oesteban) [inline mriqc/utils.py]: Normally, this kind of test is preferred as:

```
if taskinfo is None:
    ...
```

Comment (wtriplett) [inline mriqc/utils.py]: @oesteban thanks! I updated that line and also make a slight improvement to deal with non .gz image files.

**Comment by oesteban** [comment]
Once tests pass, I'll merge it to master. Good job, thanks!

**Pull request merged**